flink-runtime/src/main/java/org/apache/flink/runtime/taskexecutor/TaskManagerServicesConfiguration.java

```diff
@@ -128,8 +128,8 @@ public InetAddress getTaskManagerAddress() {
 		return localRecoveryStateRootDirectories;
 	}
 
-	public LocalRecoveryConfig.LocalRecoveryMode getLocalRecoveryMode() {
-		return localRecoveryMode;
+	public boolean isLocalRecovery() {
```

Comment:
Maybe we could rename it to `isLocalRecoveryEnabled`

Reply:
👍